requirements-freeze.txt

```diff
@@ -193,3 +193,4 @@ zipp==3.11.0
 zope.event==4.5.0
 zope.interface==5.4.0
 zstandard==0.18.0
+fairlearn==0.9.0
```

Comment:
Also add `fairlearn~=0.9.0` to `setup.cfg`. Make a new `decodingtrust` section under `options.extras_require` and put it there.

Reply:
Thanks for the comment. I have revised `setup.cfg` with commit `commit 406c87f`.